packages/clappr-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@clappr/core",
-  "version": "0.11.4",
+  "version": "0.11.5",
```

Comment:
I think it'd be best if we bumped `clappr/core`'s version after merging this MR.